src/middlewares/likes.middleware.js

```diff
@@ -6,10 +6,6 @@ const util = new Util();
 export default async (req, res, next) => {
   try {
     const user = await models.user.findOne({ where: { email: req.auth.email } });
-    if (!user) {
```

Comment:
Could you please add back this code. The application will run crash if you access properties of a user that doesn't exist

Reply:
> Could you please add back this code. The application will run crash if you access properties of a user that doesn't exist

uhhh okay, on it

Reply:
> Could you please add back this code. The application will run crash if you access properties of a user that doesn't exist

fixed